pipelines/cellranger/cellranger.wdl

```diff
@@ -0,0 +1,92 @@
+workflow cellranger {
+    meta: {
+        description: "Analyze 3' single-cell RNA-seq data using the 10X Genomics Cellranger pipeline."
+    }
+
+    String sample_id
+    Array[File] fastqs
+    String reference_name
+    File transcriptome_tar_gz
+    Int? expect_cells
+    String disk_space
+
+    parameter_meta {
+        sample_id: "Name of sample to run CellRanger count on"
+        fastqs: "Array of fastq files for running decoupled"
+        reference_name: "Reference name for count"
+        transcriptome_tar_gz: "Cellranger transcriptome reference"
+        expect_cells: "Expected number of recovered cells (defaults to 3000)"
+        disk_space: "Amount of disk space (GB) to allocate to the VM"
+    }
+
+    call CellRanger {
+        input:
+        sample_id = sample_id,
+        fastqs = fastqs,
+        reference = reference_name,
+        transcriptome_tar_gz = transcriptome_tar_gz,
+        expect_cells = expect_cells,
+        disk_space = disk_space
+   }
+}
+
+task CellRanger {
+    String sample_id
+    Array[File] fastqs
+    String reference
+    File transcriptome_tar_gz
+    Int? expect_cells
+    String disk_space
+
+    command {
+        set -e
+        mkdir transcriptome_dir
+        tar xf ${transcriptome_tar_gz} -C transcriptome_dir --strip-components 1
+        ln -s /usr/bin/python3 python
```

Comment:
Should this be done in the Dockerfile instead?

Reply:
I'm wondering if we can actually remove those lines, since Cellranger lists python 2.7.13 under its [build dependencies](https://github.com/10XGenomics/cellranger#build-dependencies) and that is the python version installed in the Dockerfile. Tagging @TimothyTickle for some clarification here, since he originally put this together!

Reply:
Is this just for the python heredoc?

Reply:
This makes the python code in the `command` block use python3, but nothing in the code requires that so I'll remove this.